app/components/Ui/Input.vue

```diff
@@ -13,14 +19,40 @@
       disabled?: boolean;
       required?: boolean;
       type?: string;
-      modelValue?: any;
+      modelValue?: string;
```

Comment:
I don't think we should restrict this to string. What about numbers?

Reply:
Ah, yeah my bad.

Reply:
Gonna merge it and then add some examples before.
Thank you btw :)